packages/web/public/static/locales/en/applications.json

```diff
@@ -10,7 +10,7 @@
     "If you and/or your organization would like to learn more about our cash transfer mechanisms, please reach out.",
   "socialPaymentsTitle": "Social Payments",
   "socialPaymentsText1":
-    "In the US, apps like Venmo and Square’s Cash app make it easy to pay a friend back for lunch. In many parts of the world, a similar experience does not exist in the local market. Celo makes it easy to send fiat-pegged digital payments to any mobile phone anywhere, in a censorship resistant manner.",
+    "In the US, apps like Venmo and Square’s Cash app make it easy to pay a friend back for lunch. In many parts of the world, a similar experience does not exist in the local market. Celo makes it easy to send fiat-tracking digital payments to any mobile phone anywhere, in a censorship resistant manner.",
```

Comment:
im unsure about how to word this one. 'fiat-pegged' fiat-tracking connotates something different. maybe it should be fiat-value tracked ?

Reply:
In the US, apps like Venmo and Square’s Cash app make it easy to pay a friend back for lunch. In many parts of the world, a similar experience does not exist in the local market. Celo makes it easy to send stable digital payments to any mobile phone anywhere, in a censorship resistant manner.